src/components/Fieldset/Fieldset.test.tsx

```diff
@@ -15,6 +15,15 @@ describe("Fieldset Accessibility", () => {
     expect(await axe(container)).toHaveNoViolations();
   });
 
+  it("passes axe accessibility with id", async () => {
```

Comment:
Should this description switch with the one above that has an id being passed?

Reply:
I missed the word "no", so "with no id".